concepticondata/conceptlists/Teit-1908-693.tsv

```diff
@@ -652,7 +652,7 @@ Teit-1908-693-650	650	594	to hunt	1435	HUNT	88
 Teit-1908-693-651	651	595	to go	695	GO	88
 Teit-1908-693-652	652	596	to go home			88
 Teit-1908-693-653	653	597	to come	1446	COME	88
-Teit-1908-693-654	654	598	" to run"	1519	RUN	88
+Teit-1908-693-654	654	598	to run	519	RUN	88
```

Comment:
This is 1519, not 519, if you run `concepticon test` you will see it is an error.